_includes/header.html

```diff
@@ -83,6 +83,13 @@ <h2>{{ site.title | escape }}</h2>
             </a>
           </li>
         {%- endif -%}
+        {%- if site.stackoverflow_username -%}
+          <li>
+            <a target="_blank" href="https://stackoverflow.com/story/{{ site.stackoverflow_username| cgi_escape | escape }}" class="button button--sacnite button--round-l">
+              <i class="fa fa-stack-overflow" title="Stackoverflow link"></i>
```

Comment:
Should we change this to `"fab fa-stack-overflow"` due to upgrade of Font Awesome #65